src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerSaveImageFunctionalTest.groovy

```diff
@@ -60,28 +53,4 @@ class DockerSaveImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
         then:
         file(CONTROL_SAVED_IMAGE).size() > file(COMPRESSED_IMAGE_FILE).size()
     }
-
-    static String buildImage() {
-        """
-            import com.bmuschko.gradle.docker.tasks.image.Dockerfile
-            import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
-            import com.bmuschko.gradle.docker.tasks.image.DockerRemoveImage
-
-            task dockerfile(type: Dockerfile) {
-                from '$TEST_IMAGE_WITH_TAG'
-            }
-
-            task buildImage(type: DockerBuildImage) {
-                dependsOn dockerfile
-                inputDir = file("build/docker")
-                tags.add("${IMAGE_ID}")
-                labels = ["setup":"${IMAGE_ID}"]
-            }
-            
-            task removeImage(type: DockerRemoveImage) {
-                force = true
-                targetImageId buildImage.getImageId()
-            }
-        """
-    }
```

Comment:
Building this image was failing with the same pop that was discussed in #770 . Not sure what's up with that just yet but using the alpine image worked as expected and vetted the test properly.